text/message_registration.go

```diff
@@ -84,7 +84,7 @@ func NewRegistrationEmailWithCodeSent() *Message {
 	return &Message{
 		ID:   InfoSelfServiceRegistrationEmailWithCodeSent,
 		Type: Info,
-		Text: "An email containing a code has been sent to the email address you provided. If you have not received an email, check the spelling of the address and retry the registration.",
+		Text: "A code has been sent to the address(es) you provided. If you have not received an message, check the spelling of the address and retry the registration.",
```

Comment:
```suggestion
		Text: "A code has been sent to the address you provided. If you have not received a message, check the spelling of the address and retry the registration.",
```

When could this be more than one address?

Reply:
Registration where both email and phone number are given and both are used for otp login

Reply:
Makes sense.

Just a typo then :) 

```suggestion
		Text: "A code has been sent to the address(es) you provided. If you have not received a message, check the spelling of the address and retry the registration.",
```